cli/cmd/customer_by_app_version.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"sort"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/replicated/cli/print"
+	"github.com/replicatedhq/replicated/pkg/kotsclient"
+	"github.com/spf13/cobra"
+)
+
+func (r *runners) InitCustomersByAppVersionCommand(parent *cobra.Command) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "search-by-app-version VERSION --app APPID",
```

Comment:
is is possible to make this just a filter flag on customer ls?

For example:

`replicated customer ls --app-version 2.2.2 --app=<slug/id>`?

No new subcommand needed this way?